jetty-server/src/main/java/org/eclipse/jetty/server/session/DefaultSessionCache.java

```diff
@@ -184,4 +195,11 @@ public boolean doReplace(String id, Session oldValue, Session newValue)
     {
         return _sessions.replace(id, oldValue, newValue);
     }
+
+    @Override
+    protected void doStart() throws Exception
+    {
+        Objects.requireNonNull(_sessions, "Session Map may not be null");
```

Comment:
This check shouldn't be necessary:  `_sessions` is  `final` && the constructor at line 51 ensures a non-null map && the other constructor at line 61 rejects nulls.

Reply:
I just removed the entire `doStart()`